src/SignalR/clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -322,6 +322,7 @@ export class HttpConnection implements IConnection {
             const response = await this._httpClient.post(negotiateUrl, {
                 content: "",
                 headers: { ...headers, ...this._options.headers },
+                timeout: 100 * 1000,
```

Comment:
Configurable?

Reply:
We could, defaultHttpTimeout? Applies to every http request except long polling polls?

Reply:
option added